### DISCUSSION TITLE
Wondering if anyone else has encountered extra key support addon seems to break the key any blocks

### DISCUSSION THREAD

**Discussion by ksdavidc** [ScratchAddons/ScratchAddons#4698] [Discussion]
Wondering if anyone else has encountered this?
_[edit] woahhh...Originally I thought this was just extra key support, but now this happens even if extra key support is off. Not sure what is going on!!]_

### Description

The extra key support addon _[edit: maybe not..]_ seems to break the key any blocks (when key any, if key any...). 

I understand that the addon is in beta, and that this may be a function of my particular system, keyboard, configuration, but just in case others are having this problem, this may be a potential bug. Of course I can just turn it off...

### Reproduction steps

_[edit: I thought it was extra key support, but even with that off, I still can't get key any to work...]_

1. disable key support addon
2. check value of <key any pressed> when no keys are pressed  reads false. 
3. enable key support
4. check value as before. reads true, even when no keys are pressed

### Expected behavior

if no key is being pressed, should render as false

### Actual behavior
renders true

### Environment

- Browser: 103.0.5060.53 
- Scratch Addons version:  latest
- Operating system:  Mac

**Comment by mxmou** [comment]
This is not related to Scratch Addons. It can be reproduced consistently even in Scratch Desktop using the following steps:
* Press a key that Scratch recognizes (any character key, arrow, space, or enter).
* While holding that key, click on another window or the desktop.
* Release the key.
* Go to the Scratch window again and check the value of "key (any) pressed". It will be true until you press the same key again.
